pkgs/servers/web-apps/restya-board/default.nix

```diff
@@ -1,46 +1,53 @@
-{ stdenv, fetchurl, unzip }:
+{ stdenv, fetchurl, fetchFromGitHub, unzip }:
```

Comment:
Is `unzip` still required?

Reply:
you are right. it is no longer needed. I removed it in the next commit.